### DISCUSSION TITLE
SSR / Next JS App Directory?

### DISCUSSION THREAD

**Discussion by JS-GitRepo** [L-Blondy/tw-colors#38] [Q&A] [answered]
Does this library work with SSR / Server Components / App Directory? It looks like it probably would, but would possibly throw a hydration error as the html data would change after initial server render? I plan on giving it a try to test but figured I'd ask while I'm working on some other things. Thank you for your time!

**Comment by L-Blondy** [comment, marked as answer]
It works as usual with Next.js app directory, server components and SSR.
There is simply no JS involved at runtime

**Reply by JS-GitRepo** [reply]
Thank you much for the response, and your work on this!
